markhor_tracks/src/markhor_hw_interface.cpp

```diff
@@ -248,6 +252,19 @@ void MarkhorHWInterface::write()
 
 void MarkhorHWInterface::read(const ros::Duration& period)
 {
+
+  //{ "flipper_fl_motor_j", "flipper_fr_motor_j", "flipper_rl_motor_j", "flipper_rr_motor_j" };
```

Comment:
Necessary?

Reply:
nope, was just a reminder while coding for the order of the flippers, I'll remove it